src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1764,6 +1764,17 @@ From there, simply click on a Markdown file, PDF file or SVG icon in the File Ex
     <value>Enable Find My Mouse</value>
     <comment>"Find My Mouse" is the name of the utility.</comment>
   </data>
+  <data name="MouseUtils_FindMyMouse_ActivationMethod.Header" xml:space="preserve">
+    <value>Activation Method</value>
```

Comment:
Minor nit: lowercase 'm' on Method :)

Reply:
Thank you for the review. I've done the requested changes. PTAL :)